uitools/common/src/ImagePopupMediaItem.h

```diff
@@ -41,6 +41,8 @@ class ImagePopupMediaItem : public PopupMediaItem
 
 signals:
   void imagePopupMediaItemChanged();
+  void clickedUrl(const QUrl& url);
```

Comment:
This is used for the `linkUrl` for when an image is displayed, the user can click on it to view more information.
translates to the following image of the WebMap
![2025-03-19_12-35-08](https://github.com/user-attachments/assets/2ead43e0-6455-4a44-9af2-934d69d97324)